WcaOnRails/app/models/round.rb

```diff
@@ -19,4 +19,13 @@ class Round < ApplicationRecord
       errors.add(:format, "'#{format_id}' is not allowed for '#{event.id}'")
     end
   end
+
+  def to_wcif
+    {
+      "format" => self.format_id,
+      "timeLimit" => nil,
```

Comment:
The OO-ness looks fine!

I expect that this line will become `"timeLimit" => self.time_limit.to_wcif` when the smoke has cleared. 
Or even `timeLimit: time_limit.to_wcif`.